### PR TITLE
Enable Jupyter Notebook Upload Post Editor

### PR DESCRIPTION
**Pull request by mengting1010**
Description of changeset:
- Enable jupyter notebook upload post editor 
image
- This pr only enables the ability to upload and render a html version of jupyter notebook (if it is a ipynb file, s3 will download the file to your local instead of displaying it. todo: will try to fix it in next pr) 
image

Test Plan:
Local Dev

Reviewers:
@csharplus @JJJ000

### CODE REVIEW

File: docker/config.py [hunk 1]
```diff
@@ -69,8 +69,10 @@
 # By default, the knowledge repo offers:
 # ['debug', 'oauth2', 'bitbucket', 'github', 'google', 'ldap']
 AUTH_PROVIDERS = ["google"]
-OAUTH_GOOGLE_CLIENT_ID = "<client id>"
-OAUTH_GOOGLE_CLIENT_SECRET = "<client secret>"
+OAUTH_GOOGLE_CLIENT_ID = (
```
Comment (JJJ000) [inline docker/config.py:72]: this could cause security issue, you might need to remove it from your commit.

Comment (JJJ000) [reply docker/config.py:72]: I add this file in gitignore, somehow it does not works looks like.

**Review by JJJ000 — APPROVED** [review]

File: docker/config.py (hunk 1 above)
Comment (mengting1010) [reply docker/config.py]: removed

**Review by csharplus — APPROVED** [review]
LGTM.

**Pull request merged**